save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -1,23 +1,24 @@
 package org.cqfn.save.core
 
+import okio.Path.Companion.toPath
 import org.cqfn.save.core.config.SaveConfig
 import org.cqfn.save.core.files.ConfigDetector
 import org.cqfn.save.core.plugin.Plugin
 
 /**
- * @property saveConfig an instance of [SaveConfig]
+ * @property saveCliConfig an instance of [SaveCliConfig]
  */
 @Suppress("INLINE_CLASS_CAN_BE_USED")  // todo: remove when there are >1 constructor parameters
 class Save(
-    private val saveConfig: SaveConfig
+    private val saveCliConfig: SaveConfig
```

Comment:
Same question about config class naming. I think, at least in core it should be just `SaveConfig`

Reply:
fixed